core/src/main/java/org/apache/accumulo/core/metrics/MetricsProducer.java

```diff
@@ -605,6 +605,8 @@ public interface MetricsProducer {
 
   Logger LOG = LoggerFactory.getLogger(MetricsProducer.class);
 
+  String METRICS_SERVER_IDLE = "accumulo.server.idle";
```

Comment:
I think the class javadoc needs to be updated too. Looks like I forgot to do this in #4078.